TimberWinR/Inputs/TailFileListener.cs

```diff
@@ -103,73 +107,100 @@ select JObject.FromObject(LogsFileDatabase.LookupLogFile(f))))
 
         private void TailFileContents(string fileName, long offset, LogsFileDatabaseEntry dbe)
         {
-            using (StreamReader reader = new StreamReader(new FileStream(fileName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite)))
-            {
-                //start at the end of the file
-                long lastMaxOffset = offset;
+            const int bufSize = 16535;
```

Comment:
I picked it since average line length (for compact json) is around 4K, so that's about 4 logs worth of overlap, and its a reasonable amount, it does not imply any limits or restrictions, it's just the buffer to grab the newline.